File: src/EmployeesCatalog.Data/Data/Entities/Employee.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace EmployeesCatalog.Data.Entities
{
    /// <summary>
    /// Описывает запись о сущности "Сотрудник" в БД.
    /// </summary>
    public class Employee
    {
        /// <summary>
        /// Уникальный идентификатор экземпляра сущности.
        /// </summary>
        [Key]
        public int EmployeeId { get; set; }

        /// <summary>
        /// Имя сотрудника.
        /// </summary>
        [Required]
        public string FirstName { get; set; }

        /// <summary>
        /// Фамилия сотрудника.
        /// </summary>
        [Required]
        public string Surname { get; set; }

        /// <summary>
        /// Отчество сотрудника.
        /// </summary>
        public string Patronymic { get; set; }

        /// <summary>
        /// Электронный ящик сотрудника.
        /// </summary>
        public string Email { get; set; }

        /// <summary>
        /// Идентификатор записи о департаменте, к которому относится сотрудник.
        /// </summary>
        public int DepartmentId { get; set; }

        /// <summary>
        /// Экземпляр <see cref="Organization"/>, в котором числится сотрудник.
        /// </summary>
        [ForeignKey("DepartmentId")]
        public Department Department { get; set; }

        /// <summary>
        /// Организация, в которой работает сотрудник.
        /// </summary>
        [NotMapped]
        public Organization Organization { get; set; }
    }
}

File: src/EmployeesCatalog.Web/ClientApp/src/app/Data/Employee.ts
/**
 * Описывает клиентскую модель сущности "Сотрудник".
 */
export class Employee {
        
        /**
         * Уникальный идентификатор экземпляра сущности.
         */
        public employeeId: number;

        
        /**
         * Имя сотрудника.
         */
        public firstName: string;

        /**
         * Фамилия сотрудника.
         */
        public surname: string;

        /**
         * Отчество сотрудника.
         */
        public patronymic: string;

        /**
         * Электронный ящик сотрудника.
         */
        public email: string;

        /**
         * Идентификатор записи о департаменте, к которому относится сотрудник.
         */
        public departmentId: number;

        /**
         * Идентификатор организации, к департаменту которой относится сотрудник.
         */
        public organizationId: number;

        /**
         * Дата рождения.
         */
        public birthDate?: string;

        /**
         * Актуальность записи о сотруднике.
         */
        public isActual: boolean = false;

        /**
         * Пол сотрудника.
         */
        public sex: number = 0;
        
}
